src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/helpdoc.txt

```diff
@@ -0,0 +1,630 @@
+    --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
+
+**************************************************************************************************
+
+(env) vishrut@mac azure-cli (devtestlabs-docs) $ az lab vm -h
+
+Group
+    az lab vm: Commands to manage vm of Azure DevTest Lab.
+
+Commands:
+    apply-artifacts: Command to apply artifacts to virtual machine in Azure DevTest Lab.
+    create         : Command to create vm of in Azure DevTest Lab.
+    delete         : Delete virtual machine.
+    list           : Command to retrieve my vms from the Azure DevTest Lab.
+    show           : Get virtual machine.
+    start          : Start a virtual machine.
+    stop           : Stop a virtual machine This operation can take a while to complete.
+
+(env) vishrut@mac azure-cli (devtestlabs-docs) $ az lab vm apply-artifacts -h
+
+Command
+    az lab vm apply-artifacts: Command to apply artifacts to virtual machine in Azure DevTest Lab.
+
+Arguments
+    --lab-name          [Required]: Name of the Lab.
+    --name -n           [Required]: Name of the virtual machine.
+    --resource-group -g [Required]: Name of the lab resource group.
+    --artifacts                   : JSON encoded array of artifacts to be applied. Use @{file} to
+                                    load from a file.
+
+Global Arguments
+    --debug                       : Increase logging verbosity to show all debug logs.
+    --help -h                     : Show this help message and exit.
+    --output -o                   : Output format.  Allowed values: json, jsonc, table, tsv.
+                                    Default: json.
+    --query                       : JMESPath query string. See http://jmespath.org/ for more
+                                    information and examples.
+    --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
+
+(env) vishrut@mac azure-cli (devtestlabs-docs) $ az lab vm create -h
+
+Command
+    az lab vm create: Command to create vm of in Azure DevTest Lab.
+
+Arguments
+    --lab-name          [Required]: Name of the Lab.
+    --name -n           [Required]: Name of the virtual machine.
+    --resource-group -g [Required]: Name of the lab resource group.
+    --allow-claim                 : Indicates whether another user can take ownership of the virtual
+                                    machine.
+    --artifacts                   : JSON encoded array of artifacts to be applied. Use @{file} to
+                                    load from a file.
+    --disk-type                   : Storage type to use for virtual machine (i.e. Standard,
+                                    Premium).
+    --expiration-date             : The expiration date in UTC(YYYY-mm-dd) for VM. Ex. 2017-03-25.
+    --formula                     : Name of the formula. Use az lab formula list for available
+                                    formulas.
```

Comment:
Yes users need to supply --artifacts with JSON file with filled parameters to be applied.